dbms/src/Server/Server.cpp

```diff
@@ -562,7 +561,23 @@ int Server::main(const std::vector<std::string> & args)
 			for (auto & server : servers)
 				server->stop();
 
-			LOG_DEBUG(log, "Closed all connections.");
+			const int sleep_max_ms = 1000 * config().getInt("shutdown_wait_unfinished", 5);
```

Comment:
Lets add another log message just after stop. Something about "Closed listening sockets. Waiting for outstanding connections."